lightning/src/chain/package.rs

```diff
@@ -777,12 +777,15 @@ fn compute_fee_from_spent_amounts<F: Deref, L: Deref>(input_amounts: u64, predic
 	      L::Target: Logger,
 {
 	let mut updated_feerate = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::HighPriority) as u64;
+	assert!(updated_feerate >= FEERATE_FLOOR_SATS_PER_KW as u64);
```

Comment:
Mh, do we really want to `assert` and panic here if something goes wrong? 
Couldn't this just default to the floor, i.e.,  `updated_feerate = max(updated_feerate, FEERATE_FLOOR_SATS_PER_KW)`? 

Alternatively, if do not want to fallback, maybe use a `debug_assert`/`log_error`/`return`, so we don't crash?

Reply:
Hmm yeah I get what you mean, but if we're getting strange feerates from our estimator then maybe we'd prefer to panic instead of silently defaulting? Although, I do think we need some sort of panic message at the least which is just an extra arg here. :man_shrugging: 

Would also be keen to know if this is what we normally want with these kinds of things in LDK.

Reply:
I add the same thought if we should `assert` or `panic`. Not sure if we have yet consistent defensive programming recommendations.